### PR TITLE
#23 make nav bar

### PR DESCRIPTION
**Pull request by SivakHico**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,11 @@
     <title>Document</title>
 </head>
 <body>
-    hhhhii 
+    <div class="topnav">
```
Comment (LubenStoyanov) [inline index.html:10]: good job

File: index.html [hunk 2]
```diff
@@ -7,6 +7,11 @@
...
+        <a class="active" href="#home">Home</a>
+        <a href="#news">News</a>
+        <a href="#contact">Contact</a>
+        <a href="#about">About</a>
+    </div>
 </body>
 </html>
```
Comment (kaybee24) [inline index.html:17]: looks good to me

**Review by LubenStoyanov — APPROVED** [review]
nice

**Review by kaybee24 — APPROVED** [review]
looks good to me

**Pull request merged**